.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -28,3 +28,13 @@ implementation proposal in the referenced issues.
 -->
 
 ...
+
+#### Checklist
+<!-- Make sure that this pull request is complete. -->
+
+- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
+- [ ] Compatibility: The changes do not break existing API, database or configuration.
```

Comment:
```suggestion
- [ ] Compatibility: The changes are backwards compatible with existing API, database and configuration.
```

Reply:
I think compatibility should also cover migrations:
`The changes are backwards compatible with existing API, database and configuration. In cases where the database schema has been changed, the migrations process should work correctly.`.

Reply:
This cannot and should not be exhaustive here as a checkbox. We have a compatibility commitment, just link to that, by checking that it aligns with the compatibility commitment.